eap/tasks/main.yml

```diff
@@ -0,0 +1,47 @@
+---
+- name: Setting Up Results Var
+  ansible.builtin.set_fact:
+    assessment_result: "pass"
+# 1. Testing WildFlyServer
+- name: Testing WildFlyServer
+  block:
+    - name: Create WildFlyServer
+      kubernetes.core.k8s:
+        name: eap7-app
+        namespace: "{{ target_namespace }}"
+        api_version: wildfly.org/v1alpha1
+        kind: WildFlyServer
+        definition:
+          metadata:
+            labels:
+              opdev: opcap
+          spec:
+            applicationImage: quay.io/opdev/eap-helloworld:v1
+            replicas: 1
+    - name: Test JBoss EAP 7 on Red Hat OpenShift
+      kubernetes.core.k8s_info:
+        name: eap7-app
+        namespace: "{{ target_namespace }}"
+        api_version: wildfly.org/v1alpha1
+        kind: WildFlyServer
+        wait: true
+      register: wildfly_info
+      until:
+        - wildfly_info.resources | length > 0
+        - wildfly_info.resources[0].status is defined
+        - wildfly_info.resources[0].status.hosts is defined
+        - wildfly_info.resources[0].status.pods[0].state is defined
+        - wildfly_info.resources[0].status.pods[0].state == "ACTIVE"
+      retries: 3
+      delay: 10
+  rescue:
+    - name: Print when failed
+      ansible.builtin.debug:
+        msg: "WildFlyServer failed. Err: {{ ansible_failed_result }}"
+    - name: Register failed tests
+      ansible.builtin.set_fact:
+        assessment_result: "fail"
+# print pass or failed according to test_results
+- name: WildFlyServer Results
+  ansible.builtin.debug:
+    msg: "{% if test_result == 'pass' %} PASS {% else %} FAILED {% endif %}"
```

Comment:
Hi @makon57 , here test result shows up as undefined for me. I guess you want assessment_result is that it? And you don't necessarily need to print the result on the screen.

Reply:
Yeah, I missed this one. I concur with Alex. Keeping this is optional.

Reply:
Changed it to assessments and left it as is.